rollup-bridge-contracts/scripts/postrm.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+set -e
+
+case "$1" in
+remove | purge)
```

Comment:
I'm not sure I understand why this is needed

Reply:
Removed